Makefile

```diff
@@ -1,5 +1,5 @@
 REG=100225593120.dkr.ecr.us-east-1.amazonaws.com
-ALLIANCE_RELEASE=5.2.0
+ALLIANCE_RELEASE=5.3.0
```

Comment:
These need to come in via ENV variables and not hardcoded.

Reply:
This is overwritten by the ENV variable.

Reply:
I just set the it to 0.0.0 to be safe.

Reply:
Is the Makefile used outside of local usage? If not, what's the harm in having the latest release defined in the Makefile? As @christabone indicated, the value of this value can always be overwritten through ENV variables, and it simplifies the use of the use of the Makefile (as in, you don't always need to go in and manually change that variable for it to work).